index.html

```diff
@@ -68,49 +74,54 @@ <h2>Why?</h2>
       </section>
 
 
-      <section>
+      <section class="quote2">
         <blockquote>
           Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
         </blockquote>
-        <p>
+        <p class="author">
           Mx. Ipsum
         </p>
       </section>
 
 
-      <section>
+      <section class="team">
         <h2>Meet the team</h2>
+        
         <ul>
-          <li>
-            <img src="https://placebear.com/150/150" alt="profile for kari kiwi" />
-            <span>Kari Kiwi</span>
-            <span>CFO</span>
-          </li>
-          <li>
-            <img src="https://placebear.com/150/150" alt="profile for dan dragon" />
-            <span>Dan Dragon</span>
-            <span>CMO</span>
-          </li>
-          <li>
-            <img src="https://placebear.com/150/150" alt="profile for chris caribou" />
-            <span>Chris Caribou</span>
-            <span>CIO</span>
-          </li>
-          <li>
-            <img src="https://placebear.com/150/150" alt="profile for charles cheetah" />
-            <span>Charles Cheetah</span>
-            <span>Board Member, CTO</span>
-          </li>
-          <li>
-            <img src="https://placebear.com/150/150" alt="profile for dee dugong" />
-            <span>Dee Dugong</span>
-            <span>Startup Enthusiast</span>
-          </li>
+            <div class="first-3">
```

Comment:
I like how you separated the first 3 team members into a different class to keep them on separate lines. Was this so that the two classes stay on different lines regardless of the size of the browser?